app/views/layouts/application.html.erb

```diff
@@ -7,19 +7,40 @@
     <%= csp_meta_tag %>
 
     <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
-    <%#= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
+    <%= stylesheet_link_tag 'https://cdn.datatables.net/1.11.5/css/dataTables.bootstrap5.min.css' %>
+    <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
   </head>
-
   <body>
+    <nav class="navbar navbar-light bg-light">
+      <a class="navbar-brand" href="/">Our Logo</a>
+      <% if current_user %>
+        <form class="form-inline">
+          <%= button_to "logout", destroy_user_session_path, method: :delete, class: 'btn btn-outline-primary' %>
+        </form>
+      <% end %>
+    </nav>
     <% if notice %>
       <p class="alert alert-success"><%= notice %></p>
     <% end %>
     <% if alert %>
       <p class="alert alert-danger"><%= alert %></p>
     <% end %>
-    <% if current_user %>
-      <%= button_to "logout", destroy_user_session_path, method: :delete  %>
-    <% end %>
+
     <%= yield %>
+    <footer class="footer-07">
```

Comment:
Maybe footer and header add to separate partials?

Reply:
probably we will use one layout, so not sure whether this is necessary